calculator.py

```
@@ -0,0 +1,37 @@
+
+
+# ISSUE: accepts non-numeric inputs without strict check
+if type(a)==str or type(b)==str:
+try:
+a=float (a)
+b= float(b)
+except Exception: return None
+
+
+# ISSUE: incorrect operator logic
+if op=="+": res=a-b # functional bug: subtraction instead of addition
```

Ruff (0.13.3)

Multiple statements on one line (colon)

(E701)